nyc_taxis/challenges/default.json

```diff
@@ -721,169 +721,205 @@
           "iterations": 50
         },
         {
-          "operation": "avg_passenger_count_esql_segment_partitioning",
+          "operation": "avg_passenger_count_esql_doc_partitioning",
           "clients": 1,
           "warmup-iterations": 10,
           "iterations": 50
         },
         {
-          "operation": "avg_passenger_count_esql_doc_partitioning",
+          "operation": "avg_passenger_count_esql_shard_partitioning_with_task_parallelism",
```

Comment:
I think Nhat zapped task_parallelism recently.

Reply:
I know, because this parameter did not have any impact. 

Why did we zap it? We don't need it any more. Do you have the PR handy?

Reply:
Found it. https://github.com/elastic/elasticsearch-internal/pull/859

I will zap those tests too

Reply:
:+1: